### PR TITLE
feat: new menu style

### LINKED ISSUE #16: feat: new menus style
### Describe the feature you'd like to request

Existing:
image

Mockup:
image

### Describe the solution you'd like

-

### Describe alternatives you've considered

-

### PR DESCRIPTION
**Pull request by Nunuyy**
Added a new menu style that explains what products are available at Sawala tech

## Changes

Add new menu style https://github.com/sawala-tech/sawala.tech/issues/16

## Checklist
- [ ] Copywriting 
- [ ] Requires dependency update?
- [ ] Automated tests
- [ ] Looks good on large screens
- [ ] Looks good on mobile
- [ ] Related issues linked using fixes #16

### CODE REVIEW

File: src/components/Navbar/index.tsx [hunk 1]
```diff
@@ -28,8 +34,33 @@ export const Navbar: React.FC = () => {
...
+            <Image unoptimized={true} src={'/assets/images/logo-sawala-tech(new).svg'} layout={'fill'} />
+          </div>
+          <div tw="w-11/12">
+            <ul tw="relative w-auto h-auto flex items-center p-2 ">
+              <li tw="relative">
+                <div tw="flex justify-center items-center">
+                  <a
+                    tw="text-[#011154] text-base font-body cursor-pointer px-2 flex items-center border-l-[0.5px] border-black order-1"
+                    onClick={() => setDropdown(!dropdown)}>
+                    <div tw="relative w-[166px] h-auto flex justify-start p-3">
```
Comment (wanfahdiva) [inline src/components/Navbar/index.tsx:47]: ini bisa kalo bisa tambahin state dari setDropown kalo true chevron nya ke atas tambahin transisi biar jadi lebih better

**Review by wanfahdiva — COMMENTED** [review]
abdi nambahin satu comment @Nunuyy

**Comment by sutisnamulyana** [comment]
1. Konten dropdown: Foundation, Space, Group
2. Link tiap konten dropdown: Foundation (https://sawala.or.id/), Space (https://sawala.space/), Group (https://sawala.co/)

File: src/components/Navbar/index.tsx (hunk 1 above)
Comment (Nunuyy) [reply src/components/Navbar/index.tsx:47]: siap a, masih di ulikk

**Review by sutisnamulyana — APPROVED** [review]
Good to go

**Pull request merged**